### PR TITLE
Uninstall by removing patch call to QMM instead of using a dll backup

### PR DESCRIPTION
**Pull request by ahk1221**

### CODE REVIEW

File: QModManager/Injector.cs [hunk 1]
```diff
@@ -93,27 +91,35 @@ internal void Remove()
...
+                    if(instruction.OpCode == OpCodes.Call && instruction.Operand.ToString().Equals("System.Void QModInstaller.QModPatcher::Patch()"))
+                    {
+                        patchMethodCall = instruction;
+                    }
                 }
 
-                Console.WriteLine();
-                Console.WriteLine("Cannot uninstall, file 'Assembly-CSharp-qoriginal.dll' is missing");
-                Console.WriteLine("To uninstall, you will need to verify game contents ons steam");
+                awakeMethod.Body.GetILProcessor().Remove(patchMethodCall);
```
Comment (AlexejheroYTB) [inline QModManager/Injector.cs]: Maybe a `null` check for `patchMethodCall`?

Comment (ahk1221) [reply QModManager/Injector.cs]: Oh yeah, forgot to check for that.

File: QModManager/Injector.cs [hunk 2]
```diff
@@ -93,27 +91,36 @@ internal void Remove()
...
+                    {
+                        patchMethodCall = instruction;
+                    }
                 }
 
-                Console.WriteLine();
-                Console.WriteLine("Cannot uninstall, file 'Assembly-CSharp-qoriginal.dll' is missing");
-                Console.WriteLine("To uninstall, you will need to verify game contents ons steam");
+                if(patchMethodCall != null)
+                    awakeMethod.Body.GetILProcessor().Remove(patchMethodCall);
```
Comment (AlexejheroYTB) [inline QModManager/Injector.cs]: If its not null, you need to tell the user that the IsInjected check succeeded, but the patch method call wasn't found. (`An unexpected error has occurred.` and exit with exit code `3`)

Comment (ahk1221) [reply QModManager/Injector.cs]: There is no way for that to happen considering the IsInjected check uses the same condition that I use to get the call instruction.

Comment (ahk1221) [reply QModManager/Injector.cs]: So the null check wasn't even required, its just "there".

Comment (AlexejheroYTB) [reply QModManager/Injector.cs]: Even if its not required, its better to have it

**Pull request merged**